zebedee-cms/src/test/java/com/github/onsdigital/zebedee/util/EncryptionUtilsTest.java

```diff
@@ -73,4 +72,19 @@ public void inputStream_withSecretKey_deciphersGobbledygook() throws IOException
         assertEquals(plain, value);
     }
 
-}
\ No newline at end of file
+    @Test
+    public void givenURL_generatingChecksum_thenVerifying()
+            throws NoSuchAlgorithmException, IOException {
+        String url = "/economy/....";
```

Comment:
You shouldn't call this `url`, because this method is completely independent of where it is being used. This is adding some coupling. A better name would be simply `input`, for instance. 

he contents of the string should also be changed and made more generic, for the same reasons.